fcrepo-indexing-solr/src/main/java/org/fcrepo/camel/indexing/solr/SolrRouter.java

```diff
@@ -94,7 +95,7 @@ public void configure() throws Exception {
             .filter(not(or(header(IDENTIFIER).startsWith(simple("{{audit.container}}/")),
                     header(IDENTIFIER).isEqualTo(simple("{{audit.container}}")))))
             .to("fcrepo:{{fcrepo.baseUrl}}?preferOmit=PreferContainment")
-            .setHeader(FCREPO_TRANSFORM).xpath(hasIndexingTransform, String.class, ns)
+            .setHeader(FCREPO_TRANSFORM).xpath(hasIndexingTransformation, String.class, ns)
```

Comment:
I would have thought this would have been caught in an integration test.
Does one need to be added?